charts/netbox/templates/superuser-secrets.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
 type: kubernetes.io/basic-auth
 data:
   username: {{ .Values.superuser.name | default "admin" | b64enc | quote }}
-  password: {{ .Values.superuser.password | default (randAlphaNum 16) | b64enc | quote }}
+  password: {{ include "common.secrets.passwords.manage" (dict "secret" (include "common.secrets.name" (dict "defaultNameSuffix" "superuser" "context" $)) "key" "password" "providedValues" (list "superuser.password") "context" $) }}
```

Comment:
hah, I ran into this in NBE at some point and just went a different route, didn't have time to look into why the common stuff wasn't working :D